src/main/resources/conf/config.yml

```diff
@@ -1,22 +1,293 @@
+                type: "metricValue"
+                properties:
+                  alias: "Tuples fetched Index Scans"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "n_tup_ins"
+                type: "metricValue"
+                properties:
+                  alias: "Number of Rows Inserted"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "n_tup_upd"
+                type: "metricValue"
+                properties:
+                  alias: "Number of Rows Updated"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "n_tup_del"
+                type: "metricValue"
+                properties:
+                  alias: "Number of Rows Deleted"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "n_live_tup"
+                type: "metricValue"
+                properties:
+                  alias: "Estimated Number of Live Rows"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "n_dead_tup"
+                type: "metricValue"
+                properties:
+                  alias: "Estimated Number of Dead Rows"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "vacuum_count"
+                type: "metricValue"
+                properties:
+                  alias: "Number of Manual Vacuums Performed"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "autovacuum_count"
+                type: "metricValue"
+                properties:
+                  alias: "Number of Auto Vacuums Performed"
+                  delta: "true"
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+              - name: "tSize"
+                type: "metricValue"
+                properties:
+                  alias: "Table Size (KB)"
+                  multiplier: 0.0009765625
+                  aggregationType: "OBSERVATION"
+                  timeRollUpType: "AVERAGE"
+                  clusterRollUpType: "INDIVIDUAL"
+
```

Comment:
aggregationType, timeRollUpType and clusterRollUpType sare same for all the metrics, hope this has been verified